### PR TITLE
Instructions to setup a validator

### PR DESCRIPTION
**Pull request by anilcse**

### CODE REVIEW

**Comment by aaronc** [comment]
@anilCSE can you update the main `README.md` as part of this PR? Either instead of or in addition to `regen-1/README.md`

**Comment by anilcse** [comment]
@clevinson I've added chainflow's gentx to this PR. Can you redo `make regen-1` and confirm before merging this PR

File: README.md [hunk 1]
```diff
@@ -12,3 +12,103 @@ For pre-launch, we can ignore errors:
...
+```
+
+### Create validator (Optional)
+Note: This section is applicable for validators who wants to join post genesis time.
+
+> **IMPORTANT:** Make sure your validator node is fully synced before running this command. Otherwise your validator will start missing blocks.
+
+```sh
+regen tx staking create-validator \
+  --amount=9000000utree \
```
Comment (clevinson) [inline README.md]:
```suggestion
  --amount=9000000uregen \
```

File: README.md [hunk 2]
```diff
@@ -12,3 +12,103 @@ For pre-launch, we can ignore errors:
...
+  --pubkey=$(regen tendermint show-validator) \
+  --moniker="<your_moniker>" \
+  --chain-id=aplikigo-1 \
```
Comment (clevinson) [inline README.md]:
```suggestion
  --chain-id=regen-1 \
```

**Comment by clevinson** [comment]
Confirmed that `make regen-1` gives me the same result. Checksum verified with the genesis.json in this branch.

Thanks @anilCSE !!

**Review by clevinson — APPROVED** [review]

**Pull request merged**